models/auth/secret.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package auth
```

Comment:
I don't think the secrets should be in the `auth` package. They have nothing to do with authentication.

Reply:
But if put them as a standalone package, there will only less files there.

Reply:
I am not quite sure what you mean?

Reply:
Is that a problem? It's wrong to keep them in `auth`. `admin`, `avatar`, `foreignreference` only contain "one" file too.